src/components/BorrowerProfile/LoanUse.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+	<p class="text-h2">
+		{{ loanUseFiltered }}
+	</p>
+</template>
+
+<script>
+const loanUseFilter = require('../../plugins/loan-use-filter');
+
+export default {
+	props: {
+		loanAmount: {
+			type: String,
+			default: '',
+		},
+		use: {
+			type: String,
+			default: '',
+		},
+		status: {
+			type: String,
+			default: '',
+		},
+		name: {
+			type: String,
+			default: '',
+		},
+		borrowerCount: {
+			type: Number,
+			default: 1,
+		}
+	},
+	computed: {
+		loanUseFiltered() {
+			// eslint-disable-next-line max-len
+			return loanUseFilter(this.use, this.name, this.status, this.loanAmount, this.borrowerCount, 100);
```

Comment:
Please use a max length of 1000, so that the loan use statement does not get trimmed. Alternatively, you could not give a max length value and update the loan use filter to not trim the loan use when `loanUseMaxLength` is 0.

Reply:
Thanks Shua, I've made this update in my latest commit.

Reply:
Looks good! You'll also need to update the test for loanUseFilter, specifically [this line](https://github.com/kiva/ui/blob/master/test/unit/specs/plugins/loanUseFilter.spec.js#L47), so that it expects the full loan use message when a max length is not provided.